Packs/RemoteAccess/Integrations/RemoteAccessv2/RemoteAccessv2.py

```diff
@@ -0,0 +1,244 @@
+    stdout_str: str = stdout.read().decode()
+    std_error_str: str = std_err.read().decode()
+    if stdout_str or std_error_str:
+        outputs: Optional[List[Dict]] = [{
+            'output': stdout_str,
+            'error': std_error_str,
+            'command': command,
+            'success': not std_error_str
+        }]
+        readable_output = tableToMarkdown(f'Command {command} Outputs', outputs, removeNull=True)
+    else:
+        outputs = None
+        readable_output = f'### Command {command} Was Executed Successfully Without Any Outputs.'
+    return CommandResults(
+        outputs_prefix='RemoteAccess.Command',
+        outputs=outputs,
+        readable_output=readable_output
+    )
+
+
+def copy_to_command(ssh_client: SSHClient, args: Dict[str, Any]) -> CommandResults:
+    """
+    Executes a safe copy from Cortex XSSOAR to remote machine.
+    Args:
+        ssh_client (SSHClient): SSH client to perform the command with.
+        args (Dict[str, Any]): Cortex XSOAR arguments.
+
+    Returns:
+        (CommandResults).
+    """
+    entry_id: str = args.get('entry_id', '')
+    if not (file_path := demisto.getFilePath(entry_id).get('path', '')):
+        raise DemistoException('Could not find given entry ID path. Please assure given entry ID is correct.')
+    destination_path: str = args.get('destination_path', file_path)
+    perform_copy_command(ssh_client, file_path, destination_path, copy_to_remote=True)
+    return CommandResults(readable_output=f'### The file corresponding to entry ID: {entry_id} was copied to remote'
+                                          ' host.')
+
+
+def copy_from_command(ssh_client: SSHClient, args: Dict[str, Any]) -> Dict:
+    """
+    Executes a safe copy from remote machine to Cortex XSOAR machine.
+    Args:
+        ssh_client (SSHClient): SSH client to perform the command with.
+        args (Dict[str, Any]): Cortex XSOAR arguments.
+
+    Returns:
+        (Dict): FileResult data.
+    """
+    file_path: str = args.get('file_path', '')
+    file_name: str = args.get('file_name', os.path.basename(file_path))
+    remote_file_data = perform_copy_command(ssh_client, file_path, file_name, copy_to_remote=False)
+
+    return fileResult(file_name, remote_file_data)
+
+
+''' MAIN FUNCTION '''
+
+
+def main() -> None:
+    params = demisto.params()
+    args = demisto.args()
+    command = demisto.command()
+
+    credentials: Dict[str, Any] = params.get('credentials', {})
+    user: str = credentials.get('identifier', '')
+    password: str = credentials.get('password', '')
+
+    host_name: str = params.get('hostname', '')
+
+    ciphers: Set[str] = set(argToList(params.get('ciphers')))
+    key_algorithms: Set[str] = set(argToList(params.get('key_algorithms')))
+
+    demisto.debug(f'Command being called is {demisto.command()}')
+    if password_param := params.get('additional_password'):
+        if password_param != args.get('additional_password'):
```

Comment:
how does it work in v1?

Reply:
you can configure additional password in the instance, and somehow you have to type that password in a command to be able to run the command

Reply:
OK, let's set a quick call todat to talk about this and the TPB